tests/e2e/provider_operation_slack_cases.py

```
@@ -0,0 +1,350 @@
+
+
+async def _thread_replies_arguments(emulate_url: str) -> dict:
+    channel_id = await _channel_id(emulate_url)
+    messages = await _history(emulate_url, channel_id)
+    root = next(
+        message for message in messages if message.get("text") == THREAD_ROOT_MARKER
+    )
+    return {"conversation": channel_id, "thread": root["ts"]}
+
+
+async def _thread_replies_outcome(emulate_url: str, preview: dict) -> None:
```

Ruff (0.16.0)

[warning] Unused function argument: `emulate_url`

(ARG001)